sdk/turing/router/config/resource_request.py

```diff
@@ -0,0 +1,89 @@
+import turing.generated.models
+from turing.generated.model_utils import OpenApiModel
+from turing.router.config.common.schemas import CpuRequestSchema, MemoryRequestSchema
+
+
+class ResourceRequest:
```

Comment:
Considering that most new SDK classes you've created are used for storing data, you might want to consider the use of `dataclasses` which comes in-built with dunder methods `__init__`, `__repr__`, `__eq__` methods that will simplify the generation and storage of attributes.
```
@dataclasses.dataclass
class ResourceRequest:
    min_replica: int
    max_replica: int
    cpu_request: str
    memory_request: str
```

Reply:
That's a great idea! I've just refactored all of the SDK classes relating to the router configs as `dataclass`-es. They should now all come with the cheap `__repr__` methods that should be convenient for users.